app/src/main/java/com/github/se/polyfit/ui/screen/LoginScreen.kt

```diff
@@ -58,43 +66,60 @@ fun LoginScreen(navController: Navigation) {
       modifier = Modifier.fillMaxSize().testTag("LoginScreen"),
       color = MaterialTheme.colorScheme.background) {
         Column(
-            modifier = Modifier.padding(24.dp),
             verticalArrangement = Arrangement.Center,
             horizontalAlignment = Alignment.CenterHorizontally) {
-              Spacer(Modifier.weight(0.15f))
+              Spacer(Modifier.weight(0.2f))
+
+              Titre(Modifier.testTag("LoginTitle"), 50.sp)
```

Comment:
titre?

Reply:
Yeah my bad, it's the french word for title